src/common/kube-helpers.ts

```diff
@@ -295,13 +295,22 @@ export function validateKubeConfig(config: KubeConfig, contextName: string, vali
 
     // Validate exec command if present
     if (validateExec && user?.exec) {
-      const execCommand = user.exec["command"];
-      // check if the command is absolute or not
-      const isAbsolute = path.isAbsolute(execCommand);
-
-      // validate the exec struct in the user object, start with the command field
-      if (!commandExists.sync(execCommand)) {
-        return new ExecValidationNotFoundError(execCommand, isAbsolute);
+      try {
+        execFileSync(user.exec.command, { timeout: 1 });
```

Comment:
Could you explain how does this fix the issue? Any tests that verify correct behavior?

Reply:
Yes tests are good.

As for how this works. The basic idea is for the OS to try and find the file or command to execute if it succeed then either node will kill the program and we get the timeout error or the program exits immediately in which no error should occur.

Reply:
I get how this change works but not sure why the previous logic didn't work? Also it feels a bit risky to execute binary, why not just check if it can be executed?